tempodb/tempodb.go

```diff
@@ -142,17 +143,20 @@ func New(cfg *Config, logger log.Logger) (Reader, Writer, Compactor, error) {
 		return nil, nil, nil, err
 	}
 
-	if cfg.Memcached != nil {
-		r, w, err = memcached.New(r, w, cfg.Memcached, logger)
+	var cacheBackend cache.Client
+	var defaultCache = false
```

Comment:
nit: No reason for `defaultCache`.  just check:

```
if cachedBackend != nil {
}
```

Reply:
Oh! Nice :) 

Changed.